assets/js/components/UserMenu.js

```diff
@@ -50,14 +50,16 @@ function UserMenu() {
 
 	useEffect( () => {
 		const handleMenuClose = ( e ) => {
-			// Close the menu if the user presses the Escape key
-			// or if they click outside of the menu.
-			if (
-				( ( 'keyup' === e.type && ESCAPE === e.keyCode ) || 'mouseup' === e.type ) &&
-				! menuButtonRef.current.buttonRef.current.contains( e.target ) &&
-				! menuRef.current.menuRef.current.contains( e.target )
-			) {
-				toggleMenu( false );
+			if ( ( menuButtonRef && menuButtonRef.current ) && ( menuRef && menuRef.current ) ) {
```

Comment:
The check for the root variable should be unnecessary as this will always be an object when called, regardless of the value of `current`. If we really need it for some reason, we should be able to simplify this like so:

```suggestion
			if ( menuButtonRef?.current && menuRef?.current ) {
```

I saw something similar elsewhere so this would apply to similar use as well.

Reply:
👍